.ci/scripts/convert_tap_to_junit.sh

```diff
@@ -1,24 +1,21 @@
 #!/usr/bin/env bash
```

Comment:
We can just delete this file now, right?

Reply:
yep, it is only for local testing until I finish the PR